cfme/tests/cloud/test_provisioning.py

```diff
@@ -47,14 +46,12 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
         'first_name': 'Image',
         'last_name': 'Provisioner',
         'notes': note,
-        'instance_type': provisioning['instance_type'] if provider.type != "azure" else
-        provisioning['vm_size'],
```

Comment:
the other way to do this is to change yamls(make azure parameters common). but that should be done carefully as I'm not fully aware which tests can be affected